test/wrb.js

```diff
@@ -394,14 +392,50 @@ contract("WitnetRequestBoard", ([
   describe("estimate gas cost", async () => {
     it("anyone can estime a data request gas cost", async () => {
       // Gas price = 1
-      const maxResRe = new BN(await this.WitnetRequestBoard.ESTIMATED_REPORT_RESULT_GAS.call())
+      const maxResRe = new BN(102496)
```

Comment:
Why hardcode the number here?

Reply:
`WRB.ESTIMATED_REPORT_RESULT_GAS` constant field was actually hardcoded in the same way. It seemed to be have a temporary and very specifc use before a better estimation mechanism is adopted. Having `WRB.estimateGasCost(uint256)` is available if needed.